doc/source/cluster/kubernetes/getting-started.ipynb

```diff
@@ -244,6 +258,165 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e60e518b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "! kubectl get pods\n",
+    "\n",
+    "# NAME                                             READY   STATUS    RESTARTS   AGE\n",
+    "# deployment-ray-head-xxxxx                        1/1     Running   0          XXs\n",
+    "# deployment-ray-worker-xxxxx                      1/1     Running   0          XXs\n",
+    "# deployment-ray-worker-xxxxx                      1/1     Running   0          XXs"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "cd8e59c6",
+   "metadata": {},
+   "source": [
+    "Wait for the pods to reach the `Running` state. This may take a few minutes -- most of this time is spent downloading the Ray images. In a separate shell, you may wish to observe the pods' status in real-time with the following command:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "1afd109c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# If you're on MacOS, first `brew install watch`.\n",
+    "# Run in a separate shell:\n",
+    "! watch -n 1 kubectl get pod"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "258c4b5f",
+   "metadata": {},
+   "source": [
+    "If your pods are stuck in the `Pending` state, you can check for errors via `kubectl describe pod deployment-ray-head-xxxx-xxxxx` and ensure that your Docker resource limits are set high enough."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "4dd57a6a",
+   "metadata": {},
+   "source": [
+    "Note that in production scenarios, you will want to use larger Ray pods. In fact, it is advantageous to size each Ray pod to take up an entire Kubernetes node. See the [configuration guide](kuberay-config) for more details."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "8f61dbc3",
+   "metadata": {},
+   "source": [
+    "#### Deploying a network policy for the static Ray cluster\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "c2db0639",
+   "metadata": {},
+   "source": [
+    "If your Kubernetes has a default deny network policy for pods, you need to manually create a network policy to allow bidirectional communication among the head and worker nodes in the Ray cluster as mentioned in [the ports configurations doc](https://docs.ray.io/en/latest/ray-core/configure.html#ports-configurations).\n",
+    "Note that the statically set port ranges for the worker ports in [the Kubernetes deployment config file](https://raw.githubusercontent.com/ray-project/ray/master/doc/source/cluster/kubernetes/configs/static-ray-cluster.with-fault-tolerance.yaml) are only required when there is a corresponding network policy created for the default deny behavior. Otherwise the statically set worker port ranges are not required."
```

Comment:
I got a 404, any ideas? https://raw.githubusercontent.com/ray-project/ray/master/doc/source/cluster/kubernetes/configs/static-ray-cluster.with-fault-tolerance.yaml

Reply:
Same for the other existing doc links, this URL is hard-coded for now. It should be able to work after merging this PR.

Reply:
Ah, gotcha!